### PR TITLE
dont add subscription tag if no such field in result

### PR DESCRIPTION
**Pull request by bergerx**
Addresses #24.

Tried to keep the change as small as possible, and I'm not sure how to add test artifacts since i tested this in one of our test environments.

We need some tests for this repo but I don't like to be blocked by spending time to place a test suite to the repo for a small change.

### CODE REVIEW

**Comment by bergerx** [comment]
For adding a test artifact,
* i spent some time on it, but it's not straightforward for me to add a test framework and then adding some tests,
* also, thought about what can i put here as an artifact like some manual script output e.g.  but only thing i have is the records in our influxdb without "subscription" tag, no logs, no output.

This is why i don't have test artifact here :(

**Comment by bergerx** [comment]
@majormoses i think you missed this, could this one be realeased? It's been impacting us for a while.

**Comment by majormoses** [comment]
@bergerx I have been on vacation this week so I will try to find some time and review it. The extensions have been a pain to manage for several reasons. We have (in the last few days) transferred all the non core extensions to the plugins org and will be continued to be supported by community maintainers. The core plugins will be maintained by sensu inc engineering team. There will be a lot of benefits that we can have by applying the maturity of tooling, processes, etc of plugins to extensions now that there is a clear line of who owns what.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format located [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
 
 ## [Unreleased]
+### Changed
```
Comment (majormoses) [inline CHANGELOG.md]: `### Fixed`

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-extensions-influxdb2/versions/0.1.1

**Pull request merged**